### PR TITLE
fix phoneoff.ru #70575

### PR DESCRIPTION
**Pull request by gork7777**
https://github.com/AdguardTeam/AdguardFilters/issues/70575

### CODE REVIEW

File: AnnoyancesFilter/sections/remains.txt [hunk 1]
```diff
@@ -75,6 +75,7 @@ link.tl##.fly_head_bottom
 extramovies.tv##.theiaStickySidebar > div[id^="ai_widget"]
 prizyv.ru###text-10.widget_text
 prizyv.ru###text-9.widget_text
+phoneoff.ru##aside#secondary.widget-area.is-active
```
Comment (zloyden) [inline AnnoyancesFilter/sections/remains.txt:78]: Use ext-css here. `phoneoff.ru#?#.site-content > aside.widget-area:has(> div.widget_text > div.widget-header:contains(Реклама))`

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Comment by zloyden** [comment]
Ok, good work here:)

**Comment by gork7777** [comment]
Can you help me little for how to do css for filters?

**Comment by zloyden** [comment]
Of course, you can read about creating rules using a pseudo-class [here](https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters#extended-css-has). There is a lot of useful information.
It is also useful to look at the current filter database in the section `css_extended.txt` - there you can see what basic principles we apply when creating such rules.

The main purpose of using such rules is when basic rules are not enough. In your case, your rule is almost fine, but it's pretty tricky. To avoid false positives and make the rule more accurate, it is safer to use ext-css.

**Comment by gork7777** [comment]
Oh... it'll take me time to learn, it's too complicated

**Pull request merged**